src/main/java/org/folio/orders/utils/PoLineCommonUtil.java

```diff
@@ -290,4 +290,8 @@ private static void checkFieldsChanged(JsonPathParser oldObject, JsonPathParser
       throw new HttpException(400, error, parameters);
     }
   }
+
+  public static CompositePoLine getPoLine(CompositePurchaseOrder compositePurchaseOrder) {
```

Comment:
need to delete this method from utils. It should not be commonly usagable. If in some places we need to get first po line - it is exceptional case, but not normal situation